plugins/MantisCoreFormatting/files/syntax-highlighting/README.md

```diff
@@ -0,0 +1,75 @@
+Syntax highlighting
+===
```

Comment:
Better use only the `#` syntax for markdown headers to avoid mixing `===`, `---` and `###`